mava/configs/default_ff_isac.yaml

```diff
@@ -3,5 +3,5 @@ defaults:
   - logger: ff_isac
   - arch: anakin
   - system: sac/ff_isac
-  - network: mlp
+  - network: continuous_mlp
```

Comment:
I like the idea 🔥 For a smooth switching between discrete and continuous case can we add this to the other systems' default option:
```
- network: mlp # [mlp, continuous_mlp ]
```